pkg/tls/certificatemanagement/csr.go

```diff
@@ -92,6 +92,15 @@ func CreateCSRInitContainer(
 
 // ResolveCsrInitImage resolves the image needed for the CSR init image taking into account the specified ImageSet
 func ResolveCSRInitImage(inst *operatorv1.InstallationSpec, is *operatorv1.ImageSet) (string, error) {
+	if inst.Variant == operatorv1.TigeraSecureEnterprise {
+		return components.GetReference(
+			components.ComponentCSRInitContainerPrivate,
+			inst.Registry,
+			inst.ImagePath,
+			inst.ImagePrefix,
+			is,
+		)
+	}
```

Comment:
Don't we need an else case here to use the Calico image?

Reply:
Line 96 returns, making the `else` redundant. This is less code and I'd argue more readable.

Reply:
Oh I see :facepalm: . What you have is good.